dotcom-rendering/src/components/AppEmailSignUp.tsx

```diff
@@ -17,22 +17,18 @@ export const AppEmailSignUp = ({
 	skipToIndex,
 	...emailSignUpProps
 }: AppEmailSignupProps) => {
-	const isCompatible = useIsBridgetCompatible();
-
-	if (!isCompatible) {
-		return null;
-	}
-
 	return (
 		<InlineSkipToWrapper
 			id={`EmailSignup-skip-link-${skipToIndex}`}
 			blockDescription="newsletter promotion"
 		>
 			<EmailSignup {...emailSignUpProps}>
-				<SecureReCAPTCHASignup
-					newsletterId={emailSignUpProps.identityName}
-					successDescription={emailSignUpProps.successDescription}
-				/>
+				<Island priority="feature" defer={{ until: 'idle' }}>
```

Comment:
As we no longer need to check bridget versions in this file, we can make just the secureRecaptcha an island